group_vars/all

```diff
@@ -25,3 +25,10 @@ validate_certs: "yes"
 # env:
 # service_vip:
 # control_interface:
+
+# should mons also run osds
+mon_osd_colocation_allowed: "true"
+
+# OSDs could be run without netplugin on them
+# osd_contiv_network_include controls this behaviour
+osd_contiv_network_include: "true"
```

Comment:
> but the flip side is that the user needs to specify every capability he wants on the node. 

hmm, yeah, it is a problem if we have too many capabilities. How about we set `host-capability` to default as `compute,storage` and then let user override it to specific values? Again this is only a problem if we have too many capabilities. Another way to deal with this (when we have a lot of properties) would be to give a short-hand/pre-defined variables for common capabilities combinations. Say, for UCS B-series we can define a `common-b-series-cap` variable and set it to `storage,cap1,cap2....` and users can instead use the short-hand strings instead?

> on the topic of keeping it to storage and not getting down to mon/osd. Is there a case, a host might want to run both ceph and nfs on the same node....
>  so, do you think specifying "Storage-node" is good enough?

Franky I don't know. But from what we have in `contiv_network`, I think in a contiv-cluster it will be either ceph or nfs but not a mix, so storage should good enough. And the differentiation of starting right set of storage daemons might be better made at the `contiv_storage` role (like we do for `contiv_network` to deploy netplugin in either aci, or standalone or bgp mode). 

But to keep it extensible, may be `storage-ceph` is a better capability as you point out than just `storage`.